lib/modules/menu.js

```diff
@@ -104,7 +103,7 @@ export function onClickMenuButton(callback, isImportant) {
 function addLegacyStyling() {
 	const gearIcon = RESPrefsLink.querySelector('.gearIcon');
 	const backgroundImage = window.getComputedStyle(gearIcon).backgroundImage;
-	if (StyleTweaks.isSubredditStyleEnabled() && backgroundImage && backgroundImage !== 'none') {
```

Comment:
Note: this check existed because styleTweaks would take way too long to remove the subreddit style.

Now that that happens during loadDynamicOptions/beforeLoad, it's always removed before this code runs (in afterLoad), so `getComputedStyle` is accurate.